src/app/globals.css

```
@@ -42,6 +42,11 @@
     height: auto;
   }
 
+  .prose :where(a, p, li) {
+    overflow-wrap: break-word;
+    word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)